src/ui/layout/browse-bar.scss

```diff
@@ -0,0 +1,63 @@
+.l-browse-bar {
```

Comment:
Please remove this file altogether. These styles have been moved to a different file in  https://github.com/nasa/openmct/pull/2568#discussion_r356324689